### ISSUE TITLE
All non ASCII characters are not displayed correctly in Italian translation

### ISSUE THREAD

**Issue by musuruan** [nxengine/lang_italian#1]
Probably the files weren't converted correctly to UTF-8.

**Comment by isage** [comment]
Can you provide screenshot?

**Comment by musuruan** [comment]
Here they are.

![Schermata da 2019-11-13 10-57-47](https://user-images.githubusercontent.com/1221995/68753520-7bcf2e80-0605-11ea-9239-ea08268552b6.png)
"[...] ma non c'è nulla"

![Schermata da 2019-11-13 11-01-33](https://user-images.githubusercontent.com/1221995/68753521-7c67c500-0605-11ea-8ef0-94259c66b838.png)
"Non c'è nulla da mangiare [...]"

![Schermata da 2019-11-13 11-01-44](https://user-images.githubusercontent.com/1221995/68753523-7c67c500-0605-11ea-81b8-dd6406bf4584.png)
"...Però se sarò costretto, mi sa che lo farò davvero..."

![Schermata da 2019-11-13 11-02-02](https://user-images.githubusercontent.com/1221995/68753524-7d005b80-0605-11ea-9c0a-19c2c1637c02.png)
"[...] Non è una di noi!"

![Schermata da 2019-11-13 11-02-06](https://user-images.githubusercontent.com/1221995/68753525-7d005b80-0605-11ea-8713-585e9e6b9848.png)
"Sue è una brava persona."

![Schermata da 2019-11-13 11-02-19](https://user-images.githubusercontent.com/1221995/68753526-7d005b80-0605-11ea-8bbf-a5a59ae3e38e.png)
"Dottore, sai già che porterà via di nuovo qualcuno..."

**Comment by isage** [comment]
Thanks, I'll take a look

**Comment by isage** [comment]
Yeah. It was conversion issue, fixed now, will be included in next release.
